addons/dev-helpers/templates/form-config-pfmon.pm.tt

```diff
@@ -10,14 +10,32 @@ pfappserver::Form::Config::Pfmon::[% class %] - Web form for [% class %] pfmon t
 
 use HTML::FormHandler::Moose;
 extends 'pfappserver::Form::Config::Pfmon';
+use pf::config::pfmon qw(%ConfigPfmonDefault);
 
 [% FOREACH attrib IN attributes -%]
 has_field '[% attrib.name %]' => ( 
     type => '[% attrib.field_type %]', 
-    default => [% attrib.default %] 
 );
 [% END -%]
 
+[% FOREACH attrib IN attributes -%]
+sub default_[% attrib.name %] {
+    return $ConfigPfmonDefault{[%class%]}{[% attrib.name %]};
+};
+[% END -%]
+
+sub default_interval {
```

Comment:
Add documentation headers for these subs and to the generated code where applicable